src/Blazor.EventGridViewer.Core/Blazor.EventGridViewer.Core.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
@bospoort, this should be net5.0 like the others.  So you can verify yourself:
- Upgrade Visual Studio to the latest version (16.9.4)
- Right click on the .EventGridViewer..Core project => Properties => Target Framework => .Net 5.0

And it should change the value.

Reply:
Updated. Thanks for coaching me through this Mike!